src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1501,6 +1501,7 @@ ConvToMatmul::ConvToMatmul(Context::Ref ctx) {
         auto matched_node_transpose_in = node_to_output.at(transpose_in).get_node_shared_ptr();
         auto matched_node_transpose_out = node_to_output.at(transpose_out).get_node_shared_ptr();
         auto matched_node_multiply = node_to_output.at(multiply).get_node_shared_ptr();
+        const auto& matched_node_param2_out = uat::_(node_to_output).at_or_at(convert2, multiply);
```

Comment:
I'd call it `cvt2_or_multiply` - as it is exactly what you're looking up for in the map.

Reply:
Done